Loop/Managers/DoseMath.swift

```diff
@@ -236,6 +236,7 @@ extension Collection where Element: GlucoseValue {
         let unit = correctionRange.unit
         let sensitivityValue = sensitivity.doubleValue(for: unit)
         let suspendThresholdValue = suspendThreshold.doubleValue(for: unit)
+        let delay = TimeInterval(minutes: 10)
```

Comment:
This value should not be hard-coded. We are also using a default value (of 10m) in LoopKit; if one changes, so should the other, so they should be controlled by a settings value, or at least a single value specified in one place.

Reply:
Yup, I was going to look into how to get that value from LoopKit, but then decided to just put this out so you can quickly take a look. 

I'd prefer including the delay in the insulin model, where it really belongs (but I'm not sure I know how to do that correctly - will take a look and get back to you in the next few days).